src/Microsoft.DotNet.Build.Tasks/PackageFiles/toolruntime.targets

```diff
@@ -1,7 +1,11 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
 
-  <UsingTask TaskName="PrereleaseResolveNuGetPackageAssets" AssemblyFile="$(ToolsDir)Microsoft.DotNet.Build.Tasks.dll"/>
+  <PropertyGroup>
```

Comment:
I think this needs to be in a more core location like the Build.Common.targets itself.